patterns/stories-podcast-hero.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|60","right":"var:preset|spacing|60"}}},"backgroundColor":"accent-2","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull has-accent-2-background-color has-background" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60);padding-left:var(--wp--preset--spacing--60)">
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-columns alignwide">
+		<!-- wp:column {"width":"40%"} -->
+		<div class="wp-block-column" style="flex-basis:40%">
+					<!-- wp:image {"sizeSlug":"large","linkDestination":"none"} -->
+					<figure class="wp-block-image size-large">
+						<img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/podcast-hero.webp" alt="<?php echo esc_attr_x( 'Picture of a person', 'Alt text for hero image', 'twentytwentyfive' ); ?>"/>
+					</figure>
+					<!-- /wp:image -->		</div>
+		<!-- /wp:column -->
+
+		<!-- wp:column {"verticalAlignment":"center","width":"60%"} -->
+		<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:60%">
+			<!-- wp:heading {"fontSize":"xx-large"} -->
+			<h2 class="wp-block-heading has-xx-large-font-size"><?php echo esc_html_x( 'The Stories® Podcast', 'Podcast title', 'twentytwentyfive' ); ?></h2>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"fontSize":"medium"} -->
+			<p class="has-medium-font-size"><?php echo esc_html_x( 'Storytelling, expert analysis, and vivid descriptions. The Stories® Podcast brings history to life, making it accessible and engaging for a global audience.', 'Podcast description', 'twentytwentyfive' ); ?></p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:heading {"level":3,"fontSize":"large"} -->
+			<h3 class="wp-block-heading has-large-font-size"><?php echo esc_html_x( 'Subscribe on your favorite platform', 'Subscription heading', 'twentytwentyfive' ); ?></h3>
+			<!-- /wp:heading -->
+
+			<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"left","orientation":"horizontal","flexWrap":"wrap"},"style":{"spacing":{"blockGap":"var:preset|spacing|20"}}} -->
+			<div class="wp-block-buttons">
+				<!-- wp:button {"className":"is-style-outline","fontSize":"extra-small"} -->
```

Comment:
This font size needs to be replaced with small.

- Would all buttons have the correct size if this was set on the buttons block instead of needing to be set on each inner button? In theory the inner buttons should inherit the correct size.